app/representers/api/v1/exercise_representer.rb

```diff
@@ -25,10 +25,7 @@ class ExerciseRepresenter < Roar::Decorator
 
     collection :questions,
                class: Question,
-               decorator: lambda { |klass, *|
-                 klass.nil? || klass.stems.length > 1 ? \
-                   QuestionRepresenter : SimpleQuestionRepresenter
-               },
+               decorator: QuestionRepresenter,
```

Comment:
This is the main substantive change of this PR, the rest is fixing the fallout of this change.